### PR TITLE
CHANGELOG: add changelog for Packer 1.9.2

### PR DESCRIPTION
**Pull request by lbajolet-hashicorp**

### CODE REVIEW

**Review by JenGoldstrich — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,59 @@
...
+### BUG FIXES:
+
+* Fixed a bug with how Packer was discovering plugins: in order to load plugins, Packer
+    would recursively scan all the known directories in which we could have installed
+    plugins. This caused a lot of unnecessary directory walks, which if the storage
+    was slow (HDD, NAS, etc.) caused major slowdowns upon invocation.
+    This has been reworked so Packer only scans directories in which plugins are
+    installed through commands such as `packer init` or `packer plugins install`,
+    which considerably reduces the number of directories scanned.
+    [GH-12414](https://github.com/hashicorp/packer/pull/12414)
```
Comment (nywilken) [inline CHANGELOG.md:28]: Slight reword to remove call out of slow storage and clarify the messaging around nested plugin scanning. I don't want to give the impression that Packer will no longer check the executable or current working directories for a single plugin. 
```suggestion
 * Fixed a bug with how Packer was discovering plugins: in order to load
      plugins, Packer would recursively scan all the known directories in which
      we could have installed plugins. This caused unnecessary directory
      walks and slowdowns upon invocation. Packer will now only check
      for nested plugins within the directories used by commands such as `packer
      init`, or `packer plugins install`, or as defined in PACKER_PLUGIN_PATH.
      Refer to the [Packer's plugin directory documentation](https://developer.hashcor.com/packer/docs/configure#packer-s-plugin-directory) for details on how loading works.
      [GH-12414](https://github.com/hashicorp/packer/pull/12414)
```

**Review by nywilken — APPROVED** [review]
Looks good. I left one suggestion. But otherwise good to go.

File: CHANGELOG.md (hunk 1 above)
Comment (lbajolet-hashicorp) [reply CHANGELOG.md]: Looks good to me, I rerolled the PR with this changed.

**Pull request merged**